src/App.tsx

```diff
@@ -1,36 +1,104 @@
-import React from 'react';
+import { Component } from 'react';
+import Carousel from './components/Carousel/Carousel';
+import { Form } from './components/Form';
+
+import { CarouselProps } from './types/CarouselProps';
+import { HandleEvent } from './types/HandleEvent';
+
 import './App.scss';
-import Carousel from './components/Carousel';
 
-interface State {
-  images: string[];
-}
+const images = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
+class App extends Component<{}, CarouselProps> {
+  state: Readonly<CarouselProps> = {
+    step: '3',
+    frameSize: '3',
+    itemWidth: '130',
+    animationDuration: '1000',
+    infinite: false,
+  };
+
+  handleChangeItemWidth: HandleEvent = e => {
+    const { value } = e.currentTarget;
+
+    this.setState({
+      itemWidth: value,
+    });
+  };
+
+  handleChangeFrameSize: HandleEvent = e => {
+    const { value } = e.currentTarget;
 
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
+    this.setState({
+      frameSize: value,
+    });
+  };
+
+  handleChangeStep: HandleEvent = e => {
+    const { value } = e.currentTarget;
+
+    this.setState({
+      step: value,
+    });
+  };
+
+  handleChangeAnimationDuration:HandleEvent = e => {
+    const { value } = e.currentTarget;
+
+    this.setState({
+      animationDuration: value,
+    });
+  };
+
+  onChangeInfinite = () => {
```

Comment:
```suggestion
  handleChangeInfinite = () => {
```
keep names consistent